insights/parsers/tuned_adm.py

```diff
@@ -0,0 +1,38 @@
+"""
+Tuned-adm - command ``/usr/sbin/tuned-adm active``
+==================================================
+This parser reads the output of the ``/usr/sbin/tuned-adm active`` command and
+reads it into a simple dictionary in the ``data`` property.
+
+Sample data::
+
+    Current active profile: default
+    Service tuned: disabled, stopped
+    Service ktune: disabled, stopped
+
+Examples::
+
+    >>> 'Service tuned' in tuned_adm_active.data
+    True
+    >>> tuned_adm_active.data['Service ktune']
+    'disabled, stopped'
```

Comment:
Since you have based the parser on `LegacyItemAccess` you could also show examples without the `.data`.